css/properties/position.json

```diff
@@ -171,6 +171,7 @@
               },
               "firefox": {
                 "version_added": "59"
```

Comment:
```suggestion
                "version_added": "59",
```

Reply:
Sorry, I just didn't notice it, but thank you anyways.